api/Competition.js

```diff
@@ -6,7 +6,7 @@ var competitionRoutes = function (app, Competition) {
             .sort({status: 'descending', name: 'ascending'})
             .exec()
             .then(function (docs) {
-                res.send(docs);
+                res.header("Access-Control-Allow-Origin", "http://localhost:3000").send(docs);
             }, function (error) {
```

Comment:
ye very true, add a a piece of middleware like this to `app.js` before the other routes are defined:
````javascript
app.get('/*',function(req,res,next){
    res.header("Access-Control-Allow-Origin", "http://localhost:3000");
    next(); 
});
````